modules/homa-lite/src/tests.rs

```diff
@@ -42,74 +42,82 @@ fn mock_initialize_token_works() {
 }
 
 #[test]
-fn request_mint_works() {
+fn mint_works() {
 	ExtBuilder::default().build().execute_with(|| {
 		let amount = dollar(1000);
 
-		assert_ok!(HomaLite::set_staking_currency_cap(Origin::signed(ROOT), dollar(1_000)));
-		let current_batch = HomaLite::current_batch();
+		assert_ok!(HomaLite::set_minting_cap(
+			Origin::signed(ROOT),
+			5 * dollar(INITIAL_BALANCE)
+		));
 
 		assert_noop!(
-			HomaLite::request_mint(Origin::signed(ROOT), amount, 0),
+			HomaLite::mint(Origin::signed(ROOT), amount, 0),
 			orml_tokens::Error::<Runtime>::BalanceTooLow
 		);
 
-		assert_ok!(HomaLite::request_mint(Origin::signed(ALICE), amount, 0));
-		assert_eq!(PendingAmount::<Runtime>::get(&current_batch, &ALICE), amount);
+		// Since the exchange rate is not set, use the default 1:10 ratio
+		// liquid = (amount - MintFee) * 10 * (1 - MaxRewardPerEra)
+		let mut liquid = Permill::from_percent(90).mul((amount - MintFee::get()) * 10);
```

Comment:
Unit tests should use numbers to verify the result. Here it's just repeating the same formula.